tests/components/deconz/test_light.py

```diff
@@ -101,7 +110,7 @@ async def test_lights_and_groups(hass):
     assert "light.on_off_switch" not in gateway.deconz_ids
     assert "light.on_off_light" in gateway.deconz_ids
 
-    assert len(hass.states.async_all()) == 5
+    assert len(hass.states.async_all()) == 6
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)